src/js/messaging/components/forms/NewMessageForm.jsx

```diff
@@ -14,20 +14,42 @@ import MessageWriteGroup from '../compose/MessageWriteGroup';
 export class NewMessageForm extends React.Component {
   render() {
     const message = this.props.message;
+    let recipientsField;
 
     // Tests the subject group for errors
     const subjectError = validations.isValidSubjectLine(message.category, message.subject);
 
+    if (this.props.recipients) {
+      if (this.props.recipients.length) {
+        recipientsField = (
+          <MessageRecipient
+              errorMessage={validations.isValidRecipient(message.recipient) ? '' : composeMessage.errors.recipient}
+              cssClass="msg-recipient msg-field"
+              onValueChange={this.props.onRecipientChange}
+              options={this.props.recipients}
+              recipient={message.recipient}/>
+        );
+      } else {
+        // The list of recipients somehow came back empty.
+        recipientsField = <p>No recipients could be found.</p>;
```

Comment:
I had `recipients` be null as the case when the request comes back with an error. If it comes back with an empty array for some reason, a retry wouldn't resolve the issue, and more action would be necessary (and an official copy should be used for the wording of that case). It's just hypothetically possible; I haven't seen it actually happen.

Your suggested change makes sense. I wasn't feeling the nested if-elses. I can go with that.